_pages/dev/styling-and-page-layout/css-architecture.md

```diff
@@ -145,6 +145,8 @@ For example, you could have a storefront app that is using the Spartacus 4.3 lib
 
 ## Implementing Versioned Styles
 
+**This section is deprecated and superseded by the next section "Implementing Feature-Flagged Styles".**
+
```

Comment:
Is this section deprecated starting with the release of 2211.19, or 2211.20, or ... 2211.21? 
I might specify the version in this statement, or if the change is true since 2211.19, I might just remove this section completely, so users only see the "one realy way" of working with versioning /feature flagged styles... Not sure... 🙂

Reply:
We still keep the old API (of "versioned styles") even though it's deprecated now. So I believe the docs for it should remain too (until the API is removed).

It's superseded since we've introduced new mechanism for feature flags styling, which is 2211.21 

https://github.com/SAP/spartacus/commit/8abc4a8b4eb4baf2957264d8d192abc4231ebfe3

> <img width="445" alt="image" src="https://github.com/SAP/spartacus-docs/assets/4001059/f2f3535f-5b55-4bef-9e64-053dbf133246">

Reply:
Perfect! Thank you for clarifying 🙂